src/plugins/console/public/application/components/shortcuts_popover/shortcuts_popover.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiPopover, EuiTitle, EuiHorizontalRule, EuiFlexGroup, EuiSpacer } from '@elastic/eui';
+import { ShortcutLineFlexItem } from './shortcut_line';
+import { KEYS } from './keys';
+
+interface ShortcutsPopoverProps {
+  button: any;
+  isOpen: boolean;
+  closePopover: () => void;
+}
+
+export const ShortcutsPopover = ({ button, isOpen, closePopover }: ShortcutsPopoverProps) => {
+  return (
+    <EuiPopover
+      button={button}
+      isOpen={isOpen}
+      closePopover={closePopover}
+      anchorPosition="downRight"
+      data-test-subj="consoleShortcutsPopover"
+    >
+      <EuiTitle size="xxs">
+        <h5>Navigation shortcuts</h5>
```

Comment:
we should translate this string

Reply:
Good catch! Addressed in [472856e](https://github.com/elastic/kibana/pull/191092/commits/472856e2c32aa4525c0c88d2cdf1084d582d1c1e)